opentelemetry-sdk/src/metrics/internal/mod.rs

```diff
@@ -71,3 +88,87 @@ impl Number<f64> for f64 {
         self
     }
 }
+
+impl AtomicTracker<u64> for AtomicU64 {
+    fn add(&self, value: u64) {
+        self.fetch_add(value, Ordering::Relaxed);
+    }
+
+    fn get_value(&self) -> u64 {
+        self.load(Ordering::Relaxed)
+    }
+
+    fn get_and_reset_value(&self) -> u64 {
+        self.swap(0, Ordering::Relaxed)
+    }
+}
+
+impl AtomicallyUpdate<u64> for u64 {
+    type AtomicTracker = AtomicU64;
+
+    fn new_atomic_tracker() -> Self::AtomicTracker {
+        AtomicU64::new(0)
+    }
+}
+
+impl AtomicTracker<i64> for AtomicI64 {
+    fn add(&self, value: i64) {
+        self.fetch_add(value, Ordering::Relaxed);
+    }
+
+    fn get_value(&self) -> i64 {
+        self.load(Ordering::Relaxed)
+    }
+
+    fn get_and_reset_value(&self) -> i64 {
+        self.swap(0, Ordering::Relaxed)
+    }
+}
+
+impl AtomicallyUpdate<i64> for i64 {
+    type AtomicTracker = AtomicI64;
+
+    fn new_atomic_tracker() -> Self::AtomicTracker {
+        AtomicI64::new(0)
+    }
+}
+
+pub(crate) struct F64AtomicTracker {
```

Comment:
it maybe too much to write the inmemoryexporter test for each type, but I think we'll need to cover it somehow, given we have different implementations for different type. Maybe some unit tests in this file can help, not a blocker for this PR